install/files/pulseaudio.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=PulseAudio system server
+
+[Service]
+Type=notify
+ExecStart=pulseaudio --daemonize=no --system --realtime --log-target=journal
```

Comment:
I would add a stop and reload command.
As well as a `after` flag as the network needs to be up before.

[Take a look here](https://gist.github.com/awidegreen/6003640)

Reply:
Hmm I'm not sure, I tried a lot of solutions to get pulseaudio running stable, with different service scripts. I know this script is working as it should. 
I'm not happy to add a new method without testing it for sometime.

Reply:
I have added the restart and after network.target, those options shouldn't be a problem